cmd/controller-manager/main.go

```diff
@@ -54,6 +55,73 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const defaultNamespace = "kube-system"
+
+// ensureCRDExists to ensure CRD exists before start
+func ensureCRDExists(kubeClient kubernetes.Interface, clientSet *versioned.Clientset) error {
+	condition := func() (bool, error) {
+		var err error
+		_, err = kubeClient.CoreV1().Namespaces().Get(defaultNamespace, metav1.GetOptions{})
+		if err == nil {
```

Comment:
How about `return` as early as we can (`err != nil`)?

Reply:
other branch that I had done

Reply:
I know. I mean, fewer indentations and handling errors close to their returned may better. but this is just my preference and you can keep this.

Reply:
I know, It's only to debug with bin file not with container. so when running with bin file, we need to check the namespace whether it exists.

Reply:
OK, then please return here is `err!=nil`.

Reply:
done